GhostNetwork.Profiles.Domain/SecuritySettings/ISecuritySettingService.cs

```diff
@@ -15,16 +16,20 @@ public interface ISecuritySettingService
             SecuritySettingsSection publications,
             SecuritySettingsSection comments,
             SecuritySettingsSection profilePhoto);
+
+        Task<bool> ResolveAccess(Guid userId, Guid toUserId, string sectionName);
```

Comment:
I think assess resolving should be in separate class that use ISecuritySettingService and IRelationsService service. No need to inject IRelationsService to settings